<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gBox_telecommande">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>196</width>
      <height>327</height>
     </rect>
    </property>
    <property name="title">
     <string>Telecommande</string>
    </property>
    <widget class="QPushButton" name="btn_close">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_supp">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>250</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_0">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_9">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_resul">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>150</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_sys_telecommande">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SYS</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_dia_telecommande">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>DIA</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_pul_telecommande">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PUL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_next">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>250</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>next</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gBox_recap">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>196</width>
      <height>327</height>
     </rect>
    </property>
    <property name="title">
     <string>RÃ©capitulatif</string>
    </property>
    <widget class="QPushButton" name="btn_close_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_sys">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_dia">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_pul">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_back">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Retour</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_send">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Envoyer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_sys">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SYS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_dia">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>DIA</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pul">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PUL</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_close_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
